packages/dapp/src/utils/arweave/index.js

```diff
@@ -16,6 +16,8 @@ export const upload = async({data, contentType}) => {
     console.log(`${uploader.pctComplete}% complete, ${uploader.uploadedChunks}/${uploader.totalChunks}`);
   }
   return {
-    id: tx.id
+    id: tx.id,
+    arUrl: `ar://${tx.id}`,
+    httpsUrl: `https://arweave.net/${tx.id}`
```

Comment:
I still have a feeling we should use `https` url, just to ensure widest compatibility
but i'm too new to arweave to know for sure how widely used `ar://` links are

Reply:
This is for metadata across marketplaces, all of whom conform to either the OpenSea or Enjin lead: https://docs.opensea.io/docs/metadata-standards#ipfs-and-arweave-uris

Reply:
Though, I am happy to store both - please just use `ar://...` in the `image` field and the https in some other, additional field.

Reply:
oh i seeeee. ok sounds good! I am using the arUrl, so all good!